virtual-prestage-child.groovy

```diff
@@ -164,6 +164,11 @@ void updateDevices(targets, targetProperty = null) {
         def command = it[3];
         def colorMode = it[4];
 
+        def value = primaryDevice.currentValue(property)
+        if( value == null ) {
```

Comment:
It looked like this entire thing was a no-op if `value` was null, so just hoisted it up here to be out of the way and skip the other processing if it was going to be discarded anyway.